cpp/docs/DEVELOPER_GUIDE.md

```diff
@@ -745,6 +745,17 @@ void isolated_helper_function(...);
 
 [**Anonymous namespaces should *never* be used in a header file.**](https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file)
 
+# Deprecating and Removing Code
+
+If changes to libcudf involve breaking existing APIs, libcudf follows the following policy:
```

Comment:
Maybe there's more to say about this. One kind of breaking change is when we remove an API, and this text explains well what should be done in that case. More often, we replace an API, and it's not clear from this text at which point the new API should be added.
Also might be good to mention that the new API should be referenced in the deprecation message.

Reply:
I've rewritten this to add some more information. Let me know what you think.